test/Vault.spec.ts

```diff
@@ -789,6 +789,26 @@ describe('Vault', () => {
   });
 
   describe('sponsor', () => {
+    describe('using an underlying that charges fees on transfer like USDC', () => {
+      beforeEach(async () => {
+        await underlying.setFee(500);
+      });
```

Comment:
Since every test starts with a clean state I think it is not necessary to have before/afterEach here

Reply:
oh, yeah, I forgot that these don't run in parallel